pkg/cosign/rekor.pub

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
Want to leave a comment somewhere (not here probably) about how you got this file from Rekor, and how/when to update it?

Reply:
Added as a comment in verify.go